build.fsx

```diff
@@ -36,6 +36,7 @@ open Fake.IO.FileSystemOperators
 open Fake.IO.Globbing.Operators
 open Fake.Core.TargetOperators
 open Fake.Api
+open Newtonsoft.Json.Linq
```

Comment:
@webwarrior-ws given that FSharpLint has moved away already from Newtonsoft, let's try using System.Text.Json here in this script

Reply:
Done